README.md

```diff
@@ -1,2 +1,82 @@
-# spacenet
-Testnet for new generation of Filecoin and Interplanetary Consensus (IPC)
+![](./assets/spacenet-header.png)
+
+# Spacenet
+> A new generation Filecoin testnet.
+>
+> Made with ❤ by [ConsensusLab](https://consensuslab.world/)
+
+- Spacenet Faucet: https://consensus-shipyard.github.io/spacenet-faucet/
+- [Spacenet Genesis](./assets/genesis/spacenet.car)
+- Spacenet Bootstraps:
+  - `xxxx`
+  - `xxxx`
+
+## Why Spacenet?
+Spacenet is not _yet another_ Filecoin testnet. Its consensus layer has been modified to integrate [Mir](https://github.com/filecoin-project/mir), a distributed protocol implementation framework. The current version of Spacenet runs an implementation of the Trantor BFT consensus over Mir. 
+_And what does this mean?_ Well, by implementing a high-performant consensus we have increased the throughput of the network significantly while reducing the 30 second block time consensus to around 1 second. As you may already be aware, Filecoin recently launched its [Filecoin Virtual Machine](https://fvm.filecoin.io/) and it will soon release support for user-defined actor. This will on board a great gamut of new applications to the Filecoin network significantly increasing its load. Certain use cases may realize that they need more from Filecoin in terms of throughput and finality times, and here is were Spacenet comes in.
```

Comment:
```suggestion
_And what does this mean?_ Well, by implementing a high-performant consensus we have increased the throughput of the network significantly while reducing the 30-second block time consensus to around 1 second. As you may already be aware, Filecoin recently launched its [Filecoin Virtual Machine](https://fvm.filecoin.io/) and it will soon release support for user-defined actors. This will onboard a gamut of new applications to the Filecoin network, significantly increasing its load. Many use cases need higher throughput and tighter finality times than Filecoin can provide, and this is where Spacenet comes in.
```